### DISCUSSION TITLE
I am able to generate report and allure-report folder is available but when using allure open Allure report is not opening

### DISCUSSION THREAD

**Discussion by Jyoti2190-Test** [webdriverio/webdriverio#7907] [Q&A]
I am able to generate report and allure-report folder is available but when using allure open Allure report is not opening, its showing can't reach the page and also index.html is showing just loading 
<img width="503" alt="image" src="https://user-images.githubusercontent.com/70073904/149063333-b8edebd9-c3bc-4867-974f-ebce8a9c8154.png">
<img width="361" alt="image" src="https://user-images.githubusercontent.com/70073904/149063369-ada05050-3d98-4afe-86b1-a4ce05b6eb84.png">
<img width="361" alt="image" src="https://user-images.githubusercontent.com/70073904/149063472-d2e6dd6d-615e-47f0-8f5c-1457e5d3934e.png">

**Comment by kitoymonster** [comment]
I think we have the same issue. It just started happening to me. Not sure if it's related to jamf since mine auto-upgraded yesterday and now its not working. Anyone know how to troubleshoot this to confirm its a firewall or some other issues? Webdriverio which uses port 4723 works fine.

Files are there but allure report page is not loading. 

Allure report page
<img width="663" alt="Screen Shot 2022-03-01 at 11 39 02 AM" src="https://user-images.githubusercontent.com/18339541/156101257-8fbf0a3f-255a-4e5d-9dfd-dfbd0c06c1bb.png">

Allure results files
<img width="681" alt="Screen Shot 2022-03-01 at 11 40 24 AM" src="https://user-images.githubusercontent.com/18339541/156101235-f506954a-6612-4357-a8d9-7bb20f79704b.png">


Allure report files
<img width="364" alt="Screen Shot 2022-03-01 at 11 41 05 AM" src="https://user-images.githubusercontent.com/18339541/156101194-ec814c3b-ed83-4a2d-9a99-7e040b208d53.png">
